pkg/apis/core/types.go

```diff
@@ -4771,9 +4771,9 @@ type ResourceQuotaScope string
 
 // These are valid values for resource quota spec
 const (
-	// Match all pod objects where spec.activeDeadlineSeconds
+	// Match all pod objects where `spec.activeDeadlineSeconds >=0`
```

Comment:
I think we should only quote `spec.activeDeadlineSeconds` (applies elsewhere):

```suggestion
	// Match all pod objects where `spec.activeDeadlineSeconds` >=0
```